packages/workspace/src/schematics/preset/preset.ts

```diff
@@ -50,7 +50,8 @@ function createPreset(options: Schema): Rule {
         'application',
         {
           name: options.name,
-          style: options.style
+          style: options.style,
+          babel: true
```

Comment:
We should also set babel as default within this block:

https://github.com/nrwl/nx/blob/master/packages/react/src/schematics/ng-add/ng-add.ts#L50

So that subsequent applications in a primarily react workspace uses babel by default.